dbms/src/Common/CurrentThread.cpp

```diff
@@ -46,6 +46,12 @@ MemoryTracker * CurrentThread::getMemoryTracker()
     return &current_thread->memory_tracker;
 }
 
+Int64 & CurrentThread::getUntrackedMemory()
+{
+    /// It assumes that (current_thread != nullptr) is already checked with getMemoryTracker()
+    return current_thread->untracked_memory;
```

Comment:
I've just overheard Alexey calling this a 'delta'. Maybe we should name it something like 'memory_delta' indeed?